monitoring/opentelemetry/pom.xml

```diff
@@ -54,6 +54,11 @@
             <artifactId>vertx-junit5</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.awaitility</groupId>
```

Comment:
I don't think this should be necessary, please re-check without it as the Awaitility are transitive dependency of Quarkust Test FW and we use them everywhere (just drop this addition and run the PR without it).